node/src/chain_spec.rs

```diff
@@ -34,7 +34,7 @@ use sp_runtime::{
 	bounded_vec,
 	traits::{IdentifyAccount, Verify},
 };
-const SAFE_XCM_VERSION: u32 = xcm::v2::VERSION;
+const SAFE_XCM_VERSION: u32 = xcm::latest::VERSION;
```

Comment:
wouldn't it be better to have the fixed version here to prevent future updates of xcm to be unwillingly brought to the codebase?

Reply:
@Hounsette The latest version of XCM would only be considered safe if we have done a full upgrade and supports that version. Bringing the safe version from a particular version was the source of some tricky upgrade issues.

Reply:
@aliXsed using the change below might allow us to avoid _some_ border effects in case we have an oversight.
```suggestion
const SAFE_XCM_VERSION: u32 = xcm::v3::VERSION;
```

Reply:
Just for reference, we discussed the cons and pros of going for the latest. Since we have had other places where the xcm definitions are fetched from the latest and we don't see any particular safety issue right now based on supporting the latest xcm that has officially brought to the scope of out code base, we decided to leave this here as it is.